scripts/deploy-docs

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${RF_DEBUG}" ]]; then
+    set -x
+fi
+
+DIR="$(dirname "$0")"
+
+set -u
+
+function usage() {
+    echo -n \
+"Usage: $(basename "$0")
+Deploy Swagger spec to a S3 website bucket.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    elif [ "${1:-}" = "--dryrun" ]; then
+        echo "Syncing docs to ${RF_SWAGGER_BUCKET} (dryrun)"
```

Comment:
We may want to conditionalize the work in here to prevent it from occurring if the docs bucket environment variable doesn't exist. Right now, all of the other environment variables are documented in the `README`, but since this one really isn't related to getting a development environment up-and-running, I think it is OK to leave it out of the `README` for now, but make it so that things don't fail if the variable isn't set.